### PR TITLE
update UI v1.4

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/components/Database/DatabaseExperiments.tsx [hunk 1]
```diff
@@ -153,6 +179,145 @@ const columns = (
   },
 ]
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseExperiments.tsx:180]: When transitioning from the Cells link of a record to Cells Viewer, Cells are not filtered by the target Experiment ID.

image

File: frontend/src/components/Database/DatabaseCells.tsx [hunk 2]
```diff
@@ -111,38 +124,39 @@ const DatabaseCells = ({ user }: CellProps) => {
   const exp_id = searchParams.get('exp_id')
   const offset = searchParams.get('offset')
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseCells.tsx:125]: Sorting not working on Experiment ID, Cell ID, columns?

File: frontend/src/components/Database/DatabaseCells.tsx [hunk 3]
```diff
@@ -28,6 +28,19 @@ type CellProps = {
 }
 
 const columns = (handleOpenDialog: (value: string[]) => void) => [
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseCells.tsx:30]: Sorting not working on Experiment ID, Cell ID, columns?

Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseCells.tsx:30]: Pixcel image, graph_urls, are not shown (broken link)

image

**Review by itutu-tienday — CHANGES_REQUESTED** [review]
I have noted the points raised. Please confirm.

File: frontend/src/components/Database/DatabaseCells.tsx [hunk 4]
```diff
@@ -28,6 +28,19 @@ type CellProps = {
...
+  {
+    field: 'experiment_id',
+    headerName: 'Experiment ID',
```
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx:36]: @itutu-tienday I fixed it

File: frontend/src/components/Database/DatabaseExperiments.tsx [hunk 5]
```diff
@@ -58,8 +86,6 @@ const columns = (
     field: 'experiment_id',
     headerName: 'Experiment ID',
```
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseExperiments.tsx:85]: @itutu-tienday I fixed it

File: frontend/src/components/Database/DatabaseExperiments.tsx [hunk 6]
```diff
@@ -31,6 +33,25 @@ import {
...
+  id: number
+  fields: {
+    brain_area: string
+    cre_driver: string
+    reporter_line: string
+    imaging_depth: number
+  }
+  experiment_id: string
+  attributes: string
+  cell_image_urls: string[]
```
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseExperiments.tsx:48]: @itutu-tienday I fixed it

File: frontend/src/components/Database/DatabaseCells.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx]: @itutu-tienday I fixed it

File: frontend/src/store/slice/Database/DatabaseType.ts [hunk 7]
```diff
@@ -4,6 +4,7 @@ export type OrderBy = 'ASC' | 'DESC' | ''
 
 export type DatabaseType = {
   id: number
+  exp_id?: string
```
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/store/slice/Database/DatabaseType.ts]: @itutu-tienday I fixed it

File: frontend/src/components/Database/DatabaseExperiments.tsx (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseExperiments.tsx:195]: @itutu-tienday I fixed it

File: frontend/src/components/Database/DatabaseCells.tsx (hunk 2 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx:151]: @itutu-tienday I fixed it

File: frontend/src/components/Database/DatabaseCells.tsx (hunk 3 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx]: @itutu-tienday I fixed it

Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx]: @itutu-tienday I fixed it

**Review by itutu-tienday — APPROVED** [review]

File: frontend/src/components/Database/DatabaseExperiments.tsx [hunk 8]
```diff
@@ -132,33 +171,175 @@ const columns = (
     width: 160,
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseExperiments.tsx:171]: The thumbnail image ("thumb_url") does not seem to be displayed in the list.
-> Please display thumbnail images in the listings. (API v0.1.5)
-> Both cell image and plot image are covered.

File: frontend/src/components/Database/DatabaseCells.tsx [hunk 9]
```diff
@@ -71,9 +102,9 @@ const columns = (handleOpenDialog: (value: string[]) => void) => [
         }}
         onClick={() => params.row?.cell_image_url && handleOpenDialog([params.row.cell_image_url])}
       >
-        {params.row?.cell_image_url && (
+        {params.row?.cell_image_url?.url && (
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseCells.tsx:105]: The thumbnail image ("thumb_url") does not seem to be displayed in the list.
-> Please display thumbnail images in the listings. (API v0.1.5)
-> Both cell image and plot image are covered.

**Review by itutu-tienday — CHANGES_REQUESTED** [review]
The remarks are noted.

File: frontend/src/components/Database/DatabaseCells.tsx (hunk 9 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx]: @itutu-tienday i fixed it

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**